scripts/ci-pull-request.sh

```diff
@@ -15,5 +15,16 @@ source ./scripts/ci-login.sh
 
 ./scripts/build-site.sh preview
 ./scripts/sync-and-test-bucket.sh preview
-./scripts/publish-sentry-release.sh
-./scripts/run-pulumi.sh preview
+
+if [ -n "${SENTRY_AUTH_TOKEN:-}" ]; then
+  echo "Publishing Sentry release..."
+  ./scripts/publish-sentry-release.sh
+else
+  echo "Sentry auth token not found. Will skip publishing release to Sentry."
+fi
+
+if [ -n "${PULUMI_STACK_NAME:-}" ]; then
+  ./scripts/run-pulumi.sh preview
+else
+  echo "PULUMI_STACK_NAME is not set. Will skip running a preview."
```

Comment:
Again, I'm not sure we ever want to do this, since a merge would mean running an update, and the absence of a Pulumi preview result is an easy thing to miss. (This PR, for example, makes an infra change whose preview was skipped because PULUMI_STACK_NAME was not set.)

Reply:
> This PR, for example, makes an infra change whose preview was skipped because PULUMI_STACK_NAME was not set.

Yep. I just realized that I never mapped the `PULUMI_STACK_NAME` to the build step. I just pushed an update.